src/utils/rest-api.js

```diff
@@ -792,6 +792,20 @@ export function updateSwitchState(studyUuid, currentNodeUuid, switchId, open) {
     });
 }
 
+export function startLoadFlow(studyUuid, currentNodeUuid) {
```

Comment:
Your rest-api.js file is not up to date : a lot of functions have already been moved to other files ....